Formula/geocouch.rb

```diff
@@ -1,11 +1,10 @@
 class Geocouch < Formula
```

Comment:
I would prefer to just deleting this formula. Otherwise, it may confuse users who want to install geocouch for the first time, as it has become as a place holder. Even more, such place holder formula will definitely confuse our CI bot.

Add maintainer feedback for more discussion.